nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError, 'subclass must override this method'
```

Comment:
[OPTIONAL]
- [ ] Great job on raising the error in the `Nameable` class, however, It would be nicer to make it dynamic and not hardcoded as you just did.
You may consider saying:
```suggestion
    raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
```

Reply:
Thanks, I will